app/controllers/application_controller/filter.rb

```diff
@@ -947,7 +947,11 @@ def adv_search_build(model)
     # Restore @edit hash if it's saved in @settings
     @expkey = :expression                                               # Reset to use default expression key
     if session[:adv_search] && session[:adv_search][model.to_s]
-      @edit = copy_hash(session[:edit])
```

Comment:
@hstastna Your change here https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/filter.rb#L950 in #776 changed something that was not touched for almost 3 years, and seems to have introduced this bug.

Therefore, we need to get `@edit = copy_hash(session[:adv_search][model.to_s])` back in ASAP because I worry that there may be other latent bugs that might get reported sooner or later.